sympy/stats/crv.py

```diff
@@ -484,8 +484,8 @@ def reduce_rational_inequalities_wrap(condition, var):
     if condition.is_Relational:
         return _reduce_inequalities([[condition]], var, relational=False)
     if condition.__class__ is Or:
```

Comment:
I know this isn't code you wrote, but this should be `isinstance(condition, Or)`.

Reply:
Thanks for pointing that out. I have opened #14598 for this.